components/common/listingCart/ListingCartModal.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+  <div>
+    <Loader v-model="isLoading" :status="status" />
+    <NeoModal
+      v-model="preferencesStore.listingCartModalOpen"
+      :can-cancel="['outside', 'escape']"
+      scroll="clip"
+      class="top"
```

Comment:
`class="top"`
repeating use in other places
Move it to be the default behavior of NeoModal, with an option to change if needed, please

Reply:
removed it